content/en/users/specific/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Specific User Documentation
+linkTitle: Cannot find it?
```

Comment:
I find this title not accurate/distracting, but other than that, I am happy with the changes.

Reply:
Fixed